Doc/tutorial/datastructures.rst

```diff
@@ -142,8 +142,8 @@ Using Lists as Stacks
 
 The list methods make it very easy to use a list as a stack, where the last
 element added is the first element retrieved ("last-in, first-out").  To add an
-item to the top of the stack, use :meth:`~list.append`.  To retrieve an item from the
-top of the stack, use :meth:`~list.pop` without an explicit index.  For example::
+item to the top of the stack, use :meth:`!~list.append`.  To retrieve an item from the
+top of the stack, use :meth:`!~list.pop` without an explicit index.  For example::
```

Comment:
This change apparently broke the rendering, we don't want to display the tilde character. The exclamation mark should probably replace the tilde?

<img width="827" alt="Zrzut ekranu 2024-12-6 o 11 52 28" src="https://github.com/user-attachments/assets/5f03254a-16f6-432a-8711-02e04289c8aa">

Reply:
@m-aciek Thank you for pointing this out. I'll fix it soon.

Reply:
Thanks, it'd be good to detect this with the Sphinx Lint too. I reported it at:

https://github.com/sphinx-contrib/sphinx-lint/issues/126